node/src/neighborhood/gossip.rs

```diff
@@ -97,37 +98,61 @@ impl TryFrom<&Value> for GossipNodeRecord {
 impl GossipNodeRecord {
     fn to_human_readable(&self) -> String {
         let mut human_readable = String::new();
-        human_readable.push_str("\nGossipNodeRecord {");
+        write!(human_readable, "\nGossipNodeRecord {{").expect("write failed");
         match NodeRecordInner_0v1::try_from(self) {
             Ok(nri) => {
-                human_readable.push_str("\n\tinner: NodeRecordInner_0v1 {");
-                human_readable.push_str(&format!("\n\t\tpublic_key: {:?},", &nri.public_key));
-                human_readable.push_str(&format!("\n\t\tnode_addr_opt: {:?},", self.node_addr_opt));
-                human_readable
-                    .push_str(&format!("\n\t\tearning_wallet: {:?},", nri.earning_wallet));
-                human_readable.push_str(&format!("\n\t\trate_pack: {:?},", nri.rate_pack));
-                human_readable.push_str(&format!(
+                write!(human_readable, "\n\tinner: NodeRecordInner_0v1 {{").expect("write failed");
```

Comment:
Sorry if you feel it otherwise but I myself would write a macro similar to short_writeln!() and replace these.

Well, or another option is that you would make a tiny card about it.

Reply:
I believe it's better to have a separate card for this. So, I created this new card - https://github.com/MASQ-Project/Node-issues/issues/615